Session Replay is now in open beta (#11674)
DataDog/documentation: content/en/real_user_monitoring/_index.md

* **Analytics / Usage**: Understand who is using your application (country, device, OS), monitor individual users journeys, and analyze how users interact with your application (most common page visited, clicks, interactions, feature usage).
* **Support**: Retrieve all of the information related to one user session to troubleshoot an issue (session duration, pages visited, interactions, resources loaded, errors).

## What is Session Replay?

> Datadog's _Session Replay_ allows you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and identifies insights in your web application’s usage patterns and design pitfalls. For more information, see [Getting Started with Session Replay][1].

Reviewer: ```suggestion
Datadog's _Session Replay_ allows you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and provides insights into your web application’s usage patterns and design pitfalls. For more information, see [Getting Started with Session Replay][1].
```

> Session Replay expands your UX monitoring by allowing you to capture and visually replay the web browsing experience of your users. Used in combination with RUM performance data, Session Replay can be an asset for error identification, reproduction, and resolution, and it can also bring invaluable insights into your web application’s usage patterns and design pitfalls. See here for a [guide to getting started with Session Replay][6].

Reviewer 2: ```suggestion
Datadog's _Session Replay_ allows you to capture and visually replay the web browsing experience of your users. Combined with RUM performance data, Session Replay is beneficial for error identification, reproduction, and resolution, and identifies insights in your web application’s usage patterns and design pitfalls. For more information, see [Getting Started with Session Replay][6].
```

> <div class="alert alert-info"><p>Session Replay is currently in beta, there are no billing implications for the Session Replays during this period. At the moment, Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a>.

Reviewer 2: ```suggestion
<div class="alert alert-info"><p>Session Replay is in beta. There are no billing implications for your Session Replays during this period. Session Replay is available only on <a href="/getting_started/site/">the US1 Datadog site</a>.
```

### Integration with logs, APM, and profiler

> View your [backend traces, logs, infrastructure metrics][5] down to the exact line of code impacting your application performance, corresponding to user experiences and reported issues.

Reviewer 2: ```suggestion
View your [backend traces, logs, and infrastructure metrics][5] down to the exact line of code impacting your application performance, corresponding to user experiences and reported issues.
```
